soak/prow/scripts/soak-on-release.sh

```diff
@@ -0,0 +1,147 @@
+eval $ASSUME_COMMAND
+>&2 echo "soak-on-release.sh] [SETUP] Assumed ACK_ROLE_ARN"
+
+AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+export AWS_ACCOUNT_ID
+>&2 echo "soak-on-release.sh] [SETUP] Exported ACCOUNT_ID=$AWS_ACCOUNT_ID"
+
+aws eks update-kubeconfig --name soak-test-cluster >/dev/null
+>&2 echo "soak-on-release.sh] [INFO] Updated the kubeconfig to communicate with 'soak-test-cluster' eks cluster."
+
+export HELM_EXPERIMENTAL_OCI=1
+cd "$SERVICE_CONTROLLER_DIR"/helm
+
+# Evaluation string for yq tool for updating helm chart's values.yaml file
+export SERVICE_ACCOUNT_ANNOTATION_EVAL=".serviceAccount.annotations.\"eks.amazonaws.com/role-arn\" = \"$IRSA_ARN\""
+export METRIC_SERVICE_CREATE_EVAL=".metrics.service.create = true"
+export METRIC_SERVICE_TYPE_EVAL=".metrics.service.type = \"ClusterIP\""
+export AWS_REGION_EVAL=".aws.region = \"us-west-2\""
+export AWS_ACCOUNT_ID_EVAL=".aws.account_id = \"$AWS_ACCOUNT_ID\""
+
+yq eval "$SERVICE_ACCOUNT_ANNOTATION_EVAL" -i values.yaml \
+&& yq eval "$METRIC_SERVICE_CREATE_EVAL" -i values.yaml \
+&& yq eval "$METRIC_SERVICE_TYPE_EVAL" -i values.yaml \
+&& yq eval "$AWS_REGION_EVAL" -i values.yaml \
+&& yq eval "$AWS_ACCOUNT_ID_EVAL" -i values.yaml
+
+export CONTROLLER_CHART_RELEASE_NAME="soak-test"
+chart_name=$(helm list -f '^soak-test$' -o json | jq -r '.[]|.name')
+[[ -n $chart_name ]] && echo "Chart soak-test already exists. Uninstalling..." && helm uninstall $CONTROLLER_CHART_RELEASE_NAME
+helm install $CONTROLLER_CHART_RELEASE_NAME . >/dev/null
+>&2 echo "soak-on-release.sh] [INFO] Helm chart $CONTROLLER_CHART_RELEASE_NAME successfully installed."
+
+# Build the soak test runner image
+cd "$TEST_INFRA_DIR"/soak
+SOAK_RUNNER_IMAGE="public.ecr.aws/aws-controllers-k8s/soak:$AWS_SERVICE"
+buildah bud \
+  -q \
+  -t $SOAK_RUNNER_IMAGE \
+  --build-arg AWS_SERVICE=$AWS_SERVICE \
+  --build-arg E2E_GIT_REF=$VERSION \
+  . >/dev/null
+
+buildah push $SOAK_RUNNER_IMAGE >/dev/null
+>&2 echo "soak-on-release.sh] [INFO] Successfully built and pushed soak runner image $SOAK_RUNNER_IMAGE"
+
+# Check for already existing soak-test-runner helm chart
+export SOAK_CHART_RELEASE_NAME="soak-test-runner"
+chart_name=$(helm list -f '^soak-test-runner$' -o json | jq -r '.[]|.name')
+[[ -n $chart_name ]] \
+&& echo "soak-on-release.sh] [INFO] Chart soak-test-runner already exists. Uninstalling..." >&2 \
+&& helm uninstall $SOAK_CHART_RELEASE_NAME >/dev/null
+
+cd "$TEST_INFRA_DIR"/soak/helm/ack-soak-test
+helm install $SOAK_CHART_RELEASE_NAME . \
+    --set awsService=$AWS_SERVICE \
+    --set soak.imageRepo="public.ecr.aws/aws-controllers-k8s/soak" \
+    --set soak.imageTag=$AWS_SERVICE \
+    --set soak.startTimeEpochSeconds=$(date +%s) \
+    --set soak.durationMinutes=1440 >/dev/null
```

Comment:
Did we want to make this duration controllable via environment variable?

Reply:
Yes, I will work on making parameters configurable in next iteration.
Kumar Gaurav Sharma also provided input on how these can be ingested from configuration file during community meeting.